pkgs/tools/admin/dehydrated/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, bash, curl, openssl, makeWrapper, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "dehydrated-0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "lukas2511";
+    repo = "dehydrated";
+    rev = "ec49a4433b48a7bc8f178d06dad3f55cff24bdf3";
+    sha256 = "0prg940ykbsfb4w48bc03j5abycg8v7f9rg9x3kcva37y8ml0jsp";
+  };
+
+  buildInputs = [ makeWrapper ];
```

Comment:
I get a 

    /nix/store/wls1rsq5hp6hd3qpgdlxifj21synm7v6-stdenv/setup: line 884: wrapProgram: command not found

when I don't include this line, so it isn't (?)